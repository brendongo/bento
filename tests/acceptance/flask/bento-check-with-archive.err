Running Bento checks...


✔ 0 findings in  s
◦ Not showing 15 archived finding(s). To view, run $ bento check --show-all.
