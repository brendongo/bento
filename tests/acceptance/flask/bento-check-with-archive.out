==> Bento Summary
r2c.bandit:
  ✔ No issues detected

r2c.click:
  ✔ No issues detected

r2c.flask:
  ✔ No issues detected

